### ISSUE TITLE
URLs that contain single or double quotes autolinks without angle brackets disappear in rendered markdown files and issues

### ISSUE THREAD

**Issue by cirosantilli** [isaacs/github#507]
Live:

`'` <http://example.com/ab'cd> http://example.com/ab'cd [a](http://example.com/ab'cd)

`"` <http://example.com/ab"cd> http://example.com/ab"cd [a](http://example.com/ab"cd)

Other "weird" characters work:

`&` <http://example.com/ab&cd> http://example.com/ab&cd [a](http://example.com/ab&cd)

I think those are all valid URLs: http://stackoverflow.com/a/25570235/895245

**Comment by kenorb** [comment]
Please report that issue at [GitHub markdown repo](https://github.com/github/markup) instead, so this can be closed.

**Comment by cirosantilli** [comment]
@kenorb yeah, it is hard choice what do do with those questions that may be answered on an open source component.

I think we should keep them here as well for the following reasons:

- what if GitHub changes the component tomorrow for another one?
- since GitHub is closed source, we are never sure if they have patched or configured those components differently
 - this way people don't need to learn what GitHub is made of, they all report here

Linking to those repos, and issues created there however, is a very good idea.
